include/picongpu/plugins/radiation/nyquist_low_pass.hpp

```diff
@@ -17,13 +17,16 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
-
 #pragma once
 
 #include "calc_amplitude.hpp"
 #include "parameters.hpp"
 #include "particle.hpp"
 
+
+namespace picongpu
```

Comment:
This leads to a failure with the use of `picongpu` in the code, there `::picongpu` or the namespace should be removed should be used.